packages/lib-core/src/types/k8s.ts

```diff
@@ -0,0 +1,122 @@
+export type K8sVerb =
```

Comment:
I'm curious on reviewers thoughts here: do we want the type definitions next to the source files (IE `lib-core/src/app/k8s` versus `lib-core/src/types`)? I've also combined some of the definitions we had in `console-types` and `common-types` into this file. I can split them back out if there is disagreement.

Reply:
For now the general convention is to put types into `src/types` (within the given monorepo package) with a meaningful file name reflecting the purpose of these types.